README.md

```diff
@@ -19,27 +19,75 @@ kubectl create ns appmesh-system
 Apply the App Mesh CRDs:
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/aws/eks-charts/master/stable/appmesh-controller/crds/crds.yaml
+kubectl apply -f https://raw.githubusercontent.com/aws/eks-charts/master/stable/appmesh-manager/crds/crds.yaml
 ```
 
-Install the App Mesh CRD controller:
+Install the App Mesh controller:
```

Comment:
Either we should move this after the section about existing mesh resources or add a note before the installation that says something like "If you're already using App Mesh and have Kubernetes App Mesh Controller installed, please refer to the upgrade section". This will help minimize any pilot errors

Reply:
I will move the note before the installation command